lib/linguist/languages.yml

```diff
@@ -2945,6 +2945,14 @@ Harbour:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 156
+Hare:
+  type: programming
+  color: "#9d7424"
+  extensions:
+  - ".ha"
+  tm_scope: source.hare
```

Comment:
You're guessing 😉 This will have no effect and will not provide syntax highlighting as you've not added a syntax highlighting grammar.

Most people want support for a language to get syntax highlighting. This PR will not give you this unless you add a grammar.

Reply:
I removed the `tm_scope` line. When I copied that I didn't realize that "tm" referred to syntax highlighting.

No, I'm not fussed about syntax highlighting. I just want Codeberg to stop saying "Makefile 100%" for my Hare projects. Maybe I'll come back after Hare reaches 1.0 and add syntax highlighting.